EnglishFilter/sections/content_blocker.txt

```diff
@@ -1645,6 +1645,8 @@ sketchful.io#@#.adBanner
 @@||tileman.io^$generichide
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59200
 @@||ask4movie.co^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/79134
+@@||googletagmanager.com/gtag/js$domain=eririo.club|freshi.site|cookuve.com|srek.net|exe.app
```

Comment:
Also remove this
```
! https://github.com/AdguardTeam/AdguardFilters/issues/72650#issuecomment-771199740
@@||googletagmanager.com/gtag/js$domain=exe.app
```

Reply:
What's wrong? You told to add all those domain at the first place.

Reply:
This is an old rule that will become unnecessary. Use the search.
https://github.com/AdguardTeam/AdguardFilters/blob/master/EnglishFilter/sections/content_blocker.txt#L1382